### ISSUE TITLE
pretrained model 404

### ISSUE THREAD

**Issue by ZiYueZH** [KaiyangZhou/deep-person-reid#177]
Hi,
I found all pretrained model under the page model zoo occurred 404 error.

The error is:
Not Found
The requested URL /~kz303/deep-person-reid/model-zoo/image-models/hacnn_market_xent.zip was not found on this server.

Apache/2.2.15 (CentOS) Server at www.eecs.qmul.ac.uk Port 80

I think the server has some problem now.

**Comment by KaiyangZhou** [comment]
the server is no longer available, I will find an alternative way to put those models

**Comment by ZiYueZH** [comment]
Hi,
Can I ask for a pretrained model  for mlfn first?
mlfn | 32.5 | softmax | (256, 128) | 90.1 (74.3) | 81.1 (63.2) | 66.4 (37.2)

Thanks so much.

**Comment by KaiyangZhou** [comment]
@ZiYueZH drop me an email and I will send you

**Comment by gceboh** [comment]
@KaiyangZhou 
[SourceForge](https://sourceforge.net/create/) provides unlimited open source file upload space and unlimited bandwidth,which may be suitable for sharing pretrained models.

**Comment by ZiYueZH** [comment]
Hi,
My email is zhy466000@gmail.com.
Thanks so much @KaiyangZhou !

**Comment by KaiyangZhou** [comment]
@ZiYueZH hey, I was waiting for your email but did not receive your request. Now the models you need are back online. Please check.

**Comment by ZiYueZH** [comment]
> @ZiYueZH hey, I was waiting for your email but did not receive your request. Now the models you need are back online. Please check.

Ah, I write my email here , not in the request.
But that's so cool you make models back online. Thanks so much.
@KaiyangZhou
